app/models/pet.rb

```diff
@@ -10,4 +12,8 @@ def shelter_name
   def self.adoptable
     where(adoptable: true)
   end
+
+  def self.search(search)
+    where('name ILIKE ? OR breed ILIKE ?', "%#{search}%", "%#{search}%")
+  end
```

Comment:
You did not need to define your own `search` method, since there is one in `ApplicationRecord` that you inherit. I guess it doesn't hurt to have your own version that overrides that one, since yours is also checking against the breed.